tmva/tmva/src/PDEFoam.cxx

```diff
@@ -589,14 +589,11 @@ void TMVA::PDEFoam::Varedu(Double_t ceSum[5], Int_t &kBest, Double_t &xBest, Dou
          Double_t xMin=0.0; Double_t xMax=0.0;
          // Double loop over all pairs jLo<jUp
          for(Int_t jLo=1; jLo<=fNBin; jLo++) {
-            Double_t aswIn=0;  Double_t asswIn=0;
+            Double_t asswIn=0;
             for(Int_t jUp=jLo; jUp<=fNBin;jUp++) {
-               aswIn  +=     ((TH1D *)(*fHistEdg)[kProj])->GetBinContent(jUp);
```

Comment:
@lmoneta could you crosscheck that `aswIn` is indeed not intended to be used? I.e. that the oversight is setting it and not with using it, where at the moment some other, wrong variable is used instead?

Reply:
The variables are not used, I would however just comment them and leave in the code, since it is maybe some information that could be useful for debugging, if we remove the lines 594, 598 and 599  we will lose this.

Reply:
Done, even though I'm not really a fan of leaving random debugging code commented out...